### PR TITLE
create failing tests for #763

### PR DESCRIPTION
**Pull request by wcastand**
| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| Breaking change?  | no
| New feature?      | no
| Deprecations?     | no
| Spec compliancy?  | yes/no
| Tests added/pass? | yes (failing tests for now)
| Fixed tickets     | #763
| License           | MIT

Create failing tests for the issue #763 
Would be happy to fix this but still didn't figured it out yet :)

### CODE REVIEW

**Comment by wcastand** [comment]
I'm not sure the expected.json is the expected output. If someone can valid that, that would be awesome :)

File: test/fixtures/experimental/pipeline-operator/with-arrow-multiline/expected.json [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+                "line": 1,
+                "column": 11
+              },
+              "end": {
+                "line": 1,
+                "column": 21
+              }
+            },
+            "left": {
+              "type": "ExpressionStatement",
```
Comment (nicolo-ribaudo) [inline test/fixtures/experimental/pipeline-operator/with-arrow-multiline/expected.json]: This should just be the ArrowFunctionExpression (line 109)

File: test/fixtures/experimental/pipeline-operator/with-arrow-multiline/options.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "plugins": ["pipelineOperator"],
+  "only": true
```
Comment (nicolo-ribaudo) [inline test/fixtures/experimental/pipeline-operator/with-arrow-multiline/options.json]: Can you remove `"only": true`?

**Comment by nicolo-ribaudo** [comment]
@wcastand Do you also want to fix this bug or do you prefer that someone else takes it?

**Comment by danez** [comment]
We moved babylon back into the monorepo at https://github.com/babel/babel/tree/master/packages/babylon.

Unfortunately pull requests cannot be migrated between repositories automatically. If this PR is still valid please reopen it there.
Thanks for your effort.

**Pull request closed without merging**